docs/source/in5min.rst

```diff
@@ -0,0 +1,124 @@
+.. _in5min:
+
+PIConGPU in 5 Minutes
+=====================
+
+A guide to run, but not understand PIConGPU.
+It is aimed at existing and preconfigured high performance computing clusters (“supercomputers”, also: HPC systems).
+For more information on the individual steps please refer to the manual.
+
+This guide needs **shell access** (probably via ``ssh``) and **some auxilliary programs** (e.g. ``git``).
+Consider getting familiar with the shell and git.
+Please also read the tutorial for your local HPC cluster.
+
+We will use the following directories:
+
+- ``~/src/picongpu``: source files from github
+- ``~/picongpu.profile``: load the dependencies for your local environment
+- ``~/picongpu-projects``: scenarios to simulate
+- ``/scratch/mydir``: result data of the simulation runs
+
+Please replace them whenever appropriate.
+
+Get the source
```

Comment:
Regarding [Capitalization](https://docs.microsoft.com/en-us/style-guide/capitalization): everywhere else you are using *title-style capitalization*, but here you are using *sentence-style capitalization*.  Could you please adjust this title.
This is only a very minor^10 style issue, but perhaps you can change this.

Reply:
adressed in 476398ca